plugins/russian/renovels.ts

```diff
@@ -77,15 +80,18 @@ class ReN implements Plugin.PluginBase {
       novel.genres = tags.join(",");
     }
 
-    const all = body.content.count_chapters / 100 + 1;
+    const totalPages =
+      (body.content.branches?.[0]?.count_chapters ||
+        body.content.count_chapters) / 100;
     const branch_id = body.content.branches?.[0]?.id || body.content.id;
     const chapters: Plugin.ChapterItem[] = [];
 
-    for (let i = 0; i < all; i++) {
+    for (let page = 0; page < totalPages; page++) {
+      await delay(1000);
       const chapterResult = await fetchApi(
```

Comment:
I think you should implement Plugin.PagePlugin for this source.
reference: https://github.com/LNReader/lnreader-sources/blob/beta/plugins/vietnamese/truyenfull.ts

Reply:
1) the order of chapters will be disrupted
2) some pages will be blank due to filtering